src/calendar.js

```diff
@@ -40,11 +48,12 @@ const Day = React.createClass({
     const w = this.props.w
```

Comment:
"i" "d" and "w" are terrible prop names :(

Reply:
Yeah.... if you don't mind diverging from the original repo, I can rename them.